### PR TITLE
Helm chart installation for report portal

### PR DESCRIPTION
**Pull request by thasherwin**
Helm chart installation for report portal

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,39 @@
...
+- A `values.yaml` which exposes a few of the configuration options in the
+charts.
+- A `templates/_helpers.tpl` file which contains helper templates. 
+
+
+You can deploy this chart with `helm install ./<project folder>`. 
+
+Once it's installed please make sure that the PersistentVolumes directories are created
+
+Commando to create those folders:
```
Comment (MaksymBilenko) [inline README.md]: I suggest that this is not the best option to use hostpath.
I preffer to see helm chart with options:
- Use specific PVC for specific component
- Create a new one with some StorageClass or `default` by default

File: templates/api-service.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+kind: Service
+metadata:
+  name: api-0
+  labels: {{ include "labels" . | indent 4 }}
+spec:
+  ports:
+  - name: headless
+    port: 8080
+    protocol: TCP
+    targetPort: 8080
```
Comment (MaksymBilenko) [inline templates/api-service.yaml]: Not sure that we need targetPort if we are going to work via Ingress

File: templates/mongodb-statefulset.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+apiVersion: apps/v1beta2
```
Comment (MaksymBilenko) [inline templates/mongodb-statefulset.yaml]: I'd preffer to use mongodb as requirenment dependency `stable/modngodb` with values from this chart

**Comment by MaksymBilenko** [comment]
Overall that's a good point to start from.
I've left some comments, @thasherwin - could you please share your thougts
Thanks

**Comment by thasherwin** [comment]
@MaksymBilenko 
I will try to look at your comments later today

**Comment by thasherwin** [comment]
@MaksymBilenko
Currently i am a little busy with my project. I cannot change the code with those comments quickly. I did mention that this is release is Beta. There is still work to be done. At least this is a starting project.

**Pull request merged**